cmd/helpMessages.go

```diff
@@ -143,6 +143,20 @@ const resumeJobsCmdShortDescription = "Resume the existing job with the given jo
 const resumeJobsCmdLongDescription = `
 Resume the existing job with the given job ID.`
 
+const removeJobsCmdShortDescription = "Remove all files associated with the given job ID"
```

Comment:
@JohnRusk @zezha-msft how do you feel about removing all of the individual `const` tokens from this file and doing a `const` block instead?

ex.
```const(
    a = "test"
    b = "test2"
)```

Reply:
Maybe one day. Not a priority for this release IMHO